packages/lodestar/src/metrics/metrics/lodestar.ts

```diff
@@ -530,6 +530,11 @@ export function createLodestarMetrics(
         help: "The delay between when the validator should send the attestation and when it was received",
         labelNames: ["index", "src"],
       }),
+      unaggregatedAttestationMissed: register.gauge<"index" | "subnet">({
```

Comment:
This is not correct since it doesn't mean it was missed. A better name would be
```
unaggregatedAttestationSubmittedNoSubnetPeers
```
but you should clarify if you are counting
- no mesh peers
- no peers in the same topic